avocado/utils/network/hosts.py

```diff
@@ -58,7 +58,8 @@ def interfaces(self):
             names = run_command(cmd, self).split()
         except Exception as ex:
             raise NWException("Failed to get interfaces: {}".format(ex))
-
+        if "bonding_masters" in names:
+            names.remove("bonding_masters")
```

Comment:
Let me know if you want this in a seperate PR

Reply:
Yes, please, this is not related to this PR. And I'm not convinced that we need to remove those. Please, try to explain why are you removing this.